### DISCUSSION TITLE
unverfied commit

### DISCUSSION THREAD

**Discussion by malik672** [community/community#41548] [Repositories] [answered]
### Select Topic Area

Bug

### Body

whenever I commit from vscode, the commit ends up as unverified. I already have my gpg keys set up still can't find out why it ends up as unverfied

**Comment by spenserblack** [comment, marked as answer]
I'm assuming you mean VS Code locally, not in `github.dev`.
You may want to set `git config --global commit.gpgsign true` (and maybe `tag.gpgsign`, too) to enforce that *all* commits get signed by default. You may want to set `Git: Enable Commit Signing` in your VS Code settings, too.

More info: https://github.com/microsoft/vscode/wiki/Commit-Signing

**Reply by malik672** [reply]
not working

**Reply by malik672** [reply]
think I've seen the problem when i ran git log saw a different git mail in the terminal
